chatbot/src/test/java/chatbot/core/handlers/rivescript/RiveScriptOutputAnalyzerTest.java

```diff
@@ -10,23 +10,23 @@
 
 		Logger log = Logger.getLogger(RiveScriptOutputAnalyzerTest.class.getName());
 		
```

Comment:
This entire file is commented now. You mentioned that other bug fix by @guru5590(Pull Request #33 ) fixes this issue. Then this file can be uncommented and it should work well? Please check if all the functions can be uncommented.